### PR TITLE
Feature/improve install instructions

### PR DESCRIPTION
**Pull request by kmcelwee**
Small readme updates and typo fixes.

### CODE REVIEW

File: scripts/local_settings.cfg.sample [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # configuration settings for pemm flask app
 
 SOLR_URL = 'http://localhost:8983/solr/'
-SOLR_CORE = 'pemm_incipit9'
\ No newline at end of file
+SOLR_CORE = 'pemm_incipit'
```
Comment (rlskoeser) [inline scripts/local_settings.cfg.sample:4]: ha, thanks for catching and correcting this

File: README.md [hunk 2]
```diff
@@ -105,21 +105,45 @@ and one very simple Flask app.
 ### Setup
 
 - Recommended: create and activate a python 3.6 virtualenv:
-```
+```sh
 python3.6 -m venv pemm
 source pem/bin/activate
```
Comment (rlskoeser) [inline README.md:110]: I noticed another typo, this should be `pemm` instead of `pem`

**Review by rlskoeser — APPROVED** [review]
Updates look good to me, thanks for these improvements.

File: README.md (hunk 2 above)
Comment (kmcelwee) [reply README.md:110]: Good catch! I'll throw this fix in another branch

Comment (rlskoeser) [reply README.md:110]: @kmcelwee It's fine to just correct something this small in `develop` after you merge!

Comment (kmcelwee) [reply README.md:110]: Oh right! I was thinking of PUL, which has formal permissions that wouldn't allow something simple like this

**Pull request merged**